meta/splat.rb

```diff
@@ -0,0 +1,25 @@
+Mutest::Meta::Example.add :splat do
+  source 'foo(*bar)'
+
+  singleton_mutations
+  mutation 'foo'
+  mutation 'foo(nil)'
+  mutation 'foo(bar)'
+  mutation 'foo(self)'
+  mutation 'foo(*self)'
+end
+
+Mutest::Meta::Example.add :splat do
+  source 'a = *bar'
+
+  singleton_mutations
+  mutation 'a__mutest__ = *bar'
+  mutation 'a = nil'
+  mutation 'a = self'
+  mutation 'a = []'
+  mutation 'a = [nil]'
+  mutation 'a = [self]'
+  mutation 'a = [*self]'
+  mutation 'a = [bar]'
+  mutation s(:lvasgn, :a, s(:splat, s(:send, nil, :bar)))
```

Comment:
he was already there. I just added the `a = *bar` example because i wanted to see what we emitted for splats in assignments. I decided it was instructive / worth keeping in the meta.

As a note: we do _not_ convert `a = *b` to `a = b`, which we should.